### PR TITLE
Mobileweb be dead

### PR DESCRIPTION
**Pull request by sgtcoolguy**
**Description:**
It was deprecated. Now it is dead.

We shall miss him greatly.

### CODE REVIEW

File: apidoc/Titanium/XML/ProcessingInstruction.yml [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-    [DOM Level 2 API](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-1004215813) 
-    on Android and iOS. Exposes the 
-    [DOM Level 3 API](http://www.w3.org/TR/DOM-Level-3-Core/core.html#ID-1004215813) implementation on Mobile Web.
+summary: A way to keep processor-specific information in the text of the document. Implements the
+    [DOM Level 2 API](http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-1004215813)
+    on Android and iOS.
 since: "0.9"
 createable: false
-platforms: [android, iphone, ipad, mobileweb]
+platforms: [android, iphone, ipad, ]
```
Comment (hansemannn) [inline apidoc/Titanium/XML/ProcessingInstruction.yml]: The regex-removal may cause problems in situations like this one.

**Review by hansemannn — DISMISSED** [review]

File: apidoc/Titanium/XML/ProcessingInstruction.yml (hunk 1 above)
Comment (sgtcoolguy) [inline apidoc/Titanium/XML/ProcessingInstruction.yml]: I believe the YML is Ok with trailing commas in arrays like this, but I can fix them up.

**Review by cb1kenobi — APPROVED** [review]
Reviewed and tested. Looks good! **APPROVED**

**Comment by hansemannn** [comment]
@sgtcoolguy Do we have a Studio PR as well? And QE should validate this before merging @cb1kenobi.

**Comment by cb1kenobi** [comment]
@hansemannn I dare not merge this before QE validates this PR.

**Pull request merged**